traces

```diff
@@ -78,27 +82,61 @@ const REGEX = '/=([[:digit:]]+)>; rel="last"/';
                     $config['keepExcludedUsers'] ?? false,
                     $config['extractEmailDomain'] ?? false,
                     $config['fieldsWhitelist'] ? array_flip($config['fieldsWhitelist']) : [],
+                    $config['excludeRepositories'] ?? [],
                 ];
             })(
                 $configFile
             );
 
             $authHeaders = ['auth' => [$user, $password]];
-            $requestUri = str_replace('{repository}', $repository, GITHUB_CONTRIBUTORS_POINT);
 
             $client = new Client(['timeout' => $timeout,]);
 
-            $io->text('Loading contributors...');
-            $response = $client->get($requestUri);
+            $io->text('Loading repositories...');
+            if (substr_count($organization, '/') > 0) {
+                $orgRepositories = [$organization];
+            } else {
+                $requestUri = str_replace('{organization}', $organization, GITHUB_REPOSITORIES_POINT);
+                $response = $client->get($requestUri, $authHeaders);
+                if ('application/json; charset=utf-8' === $response->getHeader('Content-Type')[0]) {
```

Comment:
Only check if the response is OK instead, you can't trust the first `Content-Type` header, it can change

Reply:
Done